.ci/behat-test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+if [[ ${CURRENT_BRANCH} != "master" && -z ${CI_PR_URL} ]];
+then
+  echo -e "CI will only deploy to Pantheon if on the master branch or creating a pull requests.\n"
+  exit 0;
+fi
+
+# Bail if required environment varaibles are missing
+if [ -z "$TERMINUS_SITE" ] || [ -z "$TERMINUS_ENV" ]
+then
+  echo 'No test site specified. Set TERMINUS_SITE and TERMINUS_ENV.'
+  exit 1
+fi
+
+echo "::::::::::::::::::::::::::::::::::::::::::::::::"
+echo "Behat test site: $TERMINUS_SITE.$TERMINUS_ENV"
+echo "::::::::::::::::::::::::::::::::::::::::::::::::"
+echo
+
+# Create a backup before running Behat tests
+terminus -n backup:create $TERMINUS_SITE.$TERMINUS_ENV
+
+# Clear site cache
+terminus -n env:clear-cache $TERMINUS_SITE.$TERMINUS_ENV
+
+# Wake the multidev environment before running tests
+terminus -n env:wake $TERMINUS_SITE.$TERMINUS_ENV
+
+# Ping drush to start ssh with the app server
+terminus -n drush $TERMINUS_SITE.$TERMINUS_ENV -- version
+
+# Verbose mode and exit on errors
+set -ex
+
+# Start headless Chrome
+echo "\n Starting Chrome in headless mode ..."
+google-chrome-unstable --disable-gpu --headless --remote-debugging-address=0.0.0.0 --remote-debugging-port=9222 --no-sandbox </dev/null &>/dev/null &
```

Comment:
I don't think we use headless Chrome in any of the example tests. We should either add some js tests, or maybe just comment this out for folks who might want to start using it.

Reply:
I commented this out (vs removing it completely). Hopefully we'll be able to leverage it in the future and this way it is here easily for those folks who want to start using it.